tests/msbuild-times-reference/MSBuildDeviceIntegration.csv

```diff
@@ -0,0 +1,2 @@
+Build_No_Changes,Build_CSharp_Change,Build_AndroidResource_Change,Build_Designer_Change,Build_JLO_Change,Build_CSProj_Change,Build_XAML_Change,Build_XAML_Change_RefAssembly,Install_CSharp_Change,Install_XAML_Change
+3250,4000,4750,4250,9000,9500,9500,6250,6000,7500
```

Comment:
I realize this would introduce a "file format incompatibility" with `tests/apk-sizes-reference`, but this approach will result in *really long lines*, and makes it difficult to understand what time goes with which test.

Could we instead change the file format to:

```
# Comments and blank lines are ignored
# First non-comment row is human description of columns
Test Name,Test Time in ms (int)
# Data
Build_No_Changes,3250
Build_CSharp_Change,4000
...
```

`Setup()` gets nominally more complicated, but not *significantly* so:

```
class CSVParser {
	public static Dictionary<string, int> Parse (string file)
	{
		var contents = new Dictionary<string, int>();
		using (var reader = File.OpenText (file)) {
			string line;
			bool inData = false;
			while ((line = reader.ReadLine ()) != null) {
				line = line.Trim ();
				if (string.IsNullOrEmpty (line) || line.StartsWith ("#"))
					continue;
				// Special-case first line
				if (!inData) {
					var names = line.Split (',');
					if (names.Length != 2) {
						throw new ArgumentException ($"Found unexpected header `{line}`.  Expected 2 values, found {names.Length}!");
					}
					inData = true;
					continue;
				}
				var entry = line.Split (new[]{','}, 2);
				contents [entry [0]] = int.Parse (entry [1]);
			}
		}
		return contents;
	}
}
```

Reply:
Or we might want to use `=` on each line, instead of `,`...